src/features/query-bar/flowTypes.js

```diff
@@ -1,11 +1,13 @@
 // @flow
-import { AND, OR } from './constants';
+import { AND, OR, EMPTY_CONNECTOR } from './constants';
 
-export type SelectOptionType = {
+export type ConnectorType = typeof AND | typeof OR | typeof EMPTY_CONNECTOR;
+
+export type OptionType = {
     displayText: string,
-    fieldId?: string,
+    fieldId: string,
     type?: string,
-    value: string,
+    value: string | ConnectorType,
```

Comment:
Is it needed?

Reply:
It isn't needed but provides more context regarding the specific literals that value could be.